src/components/QueryEditor.tsx

```diff
@@ -494,6 +500,39 @@ const QueryEditor: typeof window.roamjs.extension.queryBuilder.QueryEditor = ({
         <style>{`.roamjs-query-custom-enabled .bp3-control.bp3-switch .bp3-control-indicator-child:first-child {
     height: 0;
 }`}</style>
+        <div>
+        {isEditingLabel ? (
+          <InputGroup
+            placeholder={"Enter Alias"}
+            value={label}
+            onChange={(e) => setLabel(e.target.value)}
+            onKeyDown={(e) => {
+              if (e.key === "Enter") {
+                updateBlock({ uid: parentUid, text: `{{query block:${label}}}` });
+                setIsEditingLabel(false);
+              }
+            }}
+            autoFocus
+            rightElement={
+              <Button
+                minimal
+                icon={"confirm"}
+                onClick={() => {
+                  updateBlock({ uid: parentUid, text: `{{query block:${label}}}` });
+                  setIsEditingLabel(false);
+                }}
+              />
+            }
+          />
+        ) : (
+          <span 
+            style={{display: "inline-block"}}
+            tabIndex={-1} 
+            onClick={() => setIsEditingLabel(true)}
+          >
+            {!!label ? label : "Query Alias"}
+          </span>
```

Comment:
> Not super happy with the default Query Alias <span>

When there's no alias, what about a span the is italicized, low opacity, lower font, that says `Click to edit alias` ?

Reply:
Yeah, not bad!

![image](https://user-images.githubusercontent.com/3792666/219732108-61305ca9-8e74-4871-af6b-f6903ebdd9b1.png)

I was thinking that if majority of users / queries won't use this feature then to move it to the menu.
But then the user would have to click once (or twice) to see the alias.

But I think the muted text is probably acceptable.